www/include/configuration/configServers/formServers.php

```diff
@@ -293,8 +329,14 @@
  * Form Rules
  */
 $form->registerRule('exist', 'callback', 'testExistence');
+$form->registerRule('testAdditionalRemoteServer', 'callback', 'testAdditionalRemoteServer');
 $form->addRule('name', _("Name is already in use"), 'exist');
 $form->addRule('name', _("The name of the poller is mandatory"), 'required');
+$form->addRule(
+    array('remote_additional_id', 'remote_id'),
+    _('To use additional Remote Servers a Master Remote Server must be selected.'),
```

Comment:
please add translated sentences / template

Reply:
the messages.pot file should be generated from all sources automatically

Reply:
I meant -> add the new sentence to the translation files. And please translate it in the french one ;)